### DISCUSSION TITLE
KillCurly extension disabler

### DISCUSSION THREAD

**Discussion by Twistiry** [3kh0/ext-remover#462] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do I renable the extensions On the kill curly or do they just turn back on when the computer turns off?

**Comment by Aka-but-nice** [comment]
@Twistiry, Yes, once you restart, your blocker will come back.

**Comment by rwg100** [comment, marked as answer]
shut down or restard your computer to reenable the extensions

**Reply by 3kh0** [reply]
Ah yes restard your computer

**Comment by itiswhatitis111** [comment]
I tried it and nothing happen/ Please help me,DOESNT WORK!
